test/Crate/Test/DBAL/Platforms/CratePlatformTest.php

```diff
@@ -318,7 +438,7 @@ public function testGenerateArraySQLDeclaration()
 
     public function testPlatformSupport() {
         $this->assertFalse($this->_platform->supportsSequences());
-        $this->assertTrue($this->_platform->supportsSchemas());
+        $this->assertFalse($this->_platform->supportsSchemas());
```

Comment:
In that case both methods need to be implemented: 
 * <code>getCreateSchemaSQL</code>
 * <code>listNamespaceNames</code>

Without that supporting schema does not make any sense.